Client/Frontend/Theme/Theme.swift

```diff
@@ -48,7 +48,7 @@ class TableViewColor {
 class ActionMenuColor {
     var foreground: UIColor { return defaultTextAndTint }
     var iPhoneBackgroundBlurStyle: UIBlurEffect.Style { return UIBlurEffect.Style.light }
-    var iPhoneBackground: UIColor { return UIColor.theme.browser.background.withAlphaComponent(0.9) }
+    var iPhoneBackground: UIColor { return UIColor.theme.browser.background }
```

Comment:
This is the only change I am confused about, with this change I don't see a blur effect anymore (with accessibility settings off / i.e. normal mode). It seems like an alpha is required to have this blur effect.

Nice work on this PR!

Reply:
I have an idea: revert this line, and set the alpha to 1 in applyTheme

Reply:
Thanks! And good idea, that seems to fix it -- pushing momentarily